[PATCH][wxDFB] Implement dummy wxUIActionSimulator operations to satisfy the linker.
 Due to src/common/uiactioncmn.cpp being compiled into the core library,
 wxDFB application wouldn't link due to unresolved symbols.
 Introduce empty implementations of the missing wxUIActionSimulator operations;
 they can be properly filled later.

Signed-off-by: Anders Larsen <al@alarsen.net>
---
 build/bakefiles/files.bkl |    1 +
 src/dfb/uiaction.cpp      |   37 +++++++++++++++++++++++++++++++++++++
 2 files changed, 38 insertions(+), 0 deletions(-)
 create mode 100644 src/dfb/uiaction.cpp

diff --git a/build/bakefiles/files.bkl b/build/bakefiles/files.bkl
index 2605b2b..4aa12ea 100644
--- a/build/bakefiles/files.bkl
+++ b/build/bakefiles/files.bkl
@@ -1976,6 +1976,7 @@ IMPORTANT: please read docs/tech/tn0016.txt before modifying this file!
     src/dfb/region.cpp
     src/dfb/settings.cpp
     src/dfb/toplevel.cpp
+    src/dfb/uiaction.cpp
     src/dfb/utils.cpp
     src/dfb/window.cpp
     src/dfb/wrapdfb.cpp
diff --git a/src/dfb/uiaction.cpp b/src/dfb/uiaction.cpp
new file mode 100644
index 0000000..ce960d6
--- /dev/null
+++ b/src/dfb/uiaction.cpp
@@ -0,0 +1,37 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        src/dfb/uiaction.cpp
+// Purpose:     wxUIActionSimulator implementation
+// Author:      Kevin Ollivier
+// Modified by:
+// Created:     2010-03-09
+// RCS-ID:      $Id$
+// Copyright:   (c) Kevin Ollivier
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#include <wx/defs.h>
+#include <wx/uiaction.h>
+
+void SendButtonEvent(int WXUNUSED(button), bool WXUNUSED(isDown))
+{
+}
+
+bool wxUIActionSimulator::MouseDown(int WXUNUSED(button))
+{
+    return true;
+}
+
+bool wxUIActionSimulator::MouseMove(long WXUNUSED(x), long WXUNUSED(y))
+{   
+    return true;
+}
+
+bool wxUIActionSimulator::MouseUp(int WXUNUSED(button))
+{
+    return true;
+}
+
+bool wxUIActionSimulator::Key(int WXUNUSED(keycode), bool WXUNUSED(isDown), bool WXUNUSED(shiftDown), bool WXUNUSED(cmdDown), bool WXUNUSED(altDown))
+{
+    return true;
+}
-- 
1.5.4.3

